src/dreyfus_fabric_cleanup.erl

```diff
@@ -35,3 +37,41 @@ active_sigs(#doc{body={Fields}}=Doc) ->
          {ok, Index} = dreyfus_index:design_doc_to_index(Doc, IndexName),
          Index#index.sig
      end || IndexName <- IndexNames].
+
+cleanup_local_doc(DbName, ActiveSigs) ->
```

Comment:
I am not sure if need this complex code to delete _local/purge-search ... docs.
I would just use the following code:

```
cleanup_local_doc(DbName, ActiveSigs) ->
   Opts = [{start_key, list_to_binary(?LOCAL_DOC_PREFIX ++ "purge-search-")},
        {end_key_gt, list_to_binary(?LOCAL_DOC_PREFIX ++ "purge-search1")}],
       % use fabric all_docs with Ops to get  all _local/purge-search docs
       {ok, LocalPDocs} = fabric:all_docs(DbName,....)
        % filter those the need to be deleted
        % use fabric update_docs for docs that need to be removed
        fabric:update_docs(

``